app/shop/shop.py

```diff
@@ -0,0 +1,28 @@
+import datetime
+from typing import Dict, Any
+
+from app.customer.customer import Customer

```

Comment:
You cannot import `Customer` here because you import Shop inside of the `app/customer/customer.py`. You could read more about cyclic imports in Python and try to resolve them by reformatting the project structure upon the next pull request.